net-misc/FORT-validator/FORT-validator-1.2.0.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools fcaps
+
+DESCRIPTION="FORT validator is an open source RPKI validator."
+HOMEPAGE="https://fortproject.net/validator?2"
+SRC_URI="https://github.com/NICMx/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+MY_PN="fort"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="caps libressl"
+
+DEPEND="
+	acct-group/fort
+	acct-user/fort
+	caps? ( sys-libs/libcap )
+	dev-libs/jansson
+	libressl?	( dev-libs/libressl:0= )
+	!libressl?	( dev-libs/openssl:0= )
+	net-misc/rsync
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	sys-devel/autoconf
+	sys-devel/automake
+"
+
+src_prepare() {
+	default
+
+	eautoreconf
+}
+
+src_install() {
+	newinitd "${FILESDIR}/${MY_PN}-initd" ${MY_PN}
+	newconfd "${FILESDIR}/${MY_PN}-confd" ${MY_PN}
+
+	emake DESTDIR="${D}" install
+	insinto /usr/share/${MY_PN}/
+	insopts -m0644 -o "${MY_PN}"
+	diropts -m0755 -o "${MY_PN}"
+	doins -r examples/tal/
+
+	dodoc -r examples/
+
+	insinto /etc/fort
+	newins "${FILESDIR}/fort-config.json" config.json
+
+	exeinto /usr/libexec/${MY_PN}
+	doexe fort_setup.sh
+}
+
+pkg_postinst() {
+	fcaps cap_net_bind_service usr/bin/fort
+
+	einfo ""
+	einfo "ARIN TAL is disabled by default because the ARIN Relying Party"
+	einfo "Agreement must be accepted beforehead. Start fort, run"
+	einfo ""
+	einfo "  su -s /bin/sh -c '${EROOT}/usr/libexec/${MY_PN}/fort_setup.sh /usr/share/${MY_PN}/tal/' fort"
```

Comment:
Personally, I don't like automagical scripts

Reply:
I don’t like so much either, but it’s provided by upstream and ARIN has a weird licence on their repo that prevents the TAL (the file describing how to fetch it and containing the public key) from being packaged with validators… Routinator has a similar thing, for example. See https://git.grifon.fr/alarig/SwordArMor-gentoo-overlay/src/branch/master/net-misc/routinator/routinator-0.6.4.ebuild#L238